infrastructures/infrastructure-openstack/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/OpenstackInfrastructure.java

```diff
@@ -100,6 +100,9 @@
     @Configurable(description = "Flavor type of OpenStack", sectionSelector = 3, important = true)
     protected String flavor = null;
 
+    @Configurable(description = "(optional) Network for the openstack instance", sectionSelector = 3)
+    protected String network = null;
```

Comment:
If I understand it well, its value is supposed to be one networkId. 
1/ Would it be more clear to change the parameter name to networkId ? (or networkName?)
2/ just wondering: would it be more proper to specify multiple networkIds ? (only if it's necessary and easy to do)